changelogs/current.yaml

```diff
@@ -19,6 +19,13 @@ removed_config_or_runtime:
 # *Normally occurs at the end of the* :ref:`deprecation period <deprecated>`
 
 new_features:
+- area: ext_proc
+  change: |
+    Added
+    :ref:`metadata <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExtProcOverrides.metadata>`
+    config API to allow extending inherited metadata from ExternalProcessor
```

Comment:
```suggestion
    config API to allow extending inherited metadata from ``ExternalProcessor``
```

Reply:
changed

Reply:
@sitano - huge apology here - after i commented i realized its a name rather than a literal - ideally link to the api doc i think

Reply:
@phlax - no worries. changed to 
```
...
config API to allow extending inherited metadata from
:ref:`ExternalProcessor.grpc_service <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.grpc_service>`
and 
:ref:`ExtProcOverrides.grpc_service <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExtProcOverrides.grpc_service>`
with the new or updated values.
```
to emphasize that the metadata can be inherited from either option.